src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/mercury/NavigationSideComponentObject.java

```diff
@@ -27,7 +27,7 @@
   @FindBy(css = ".local-wikia-search a")
   private WebElement searchSuggestion;
 
-  @FindBy(xpath = "//ul[contains(@class, \"local-nav-menu\")]/li/*")
```

Comment:
the first solution that @justnpT proposed was using just `ul.local-nav-menu li` which returns one additional selector above nav list.
Together we figured it out and used `ul.local-nav-menu li > *` which is actualy the same as previous xpath solution